tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -36,6 +36,8 @@
 MOCK_SUBDIR = "subdir/path"
 MOCK_SUBDIR_ROOT_URI = posixpath.join(MOCK_RUN_ROOT_URI, MOCK_SUBDIR)
 
+pytestmark = pytest.mark.skip
```

Comment:
Many tests in this module get stuck... @dbczumar Could you provide some guide/clue what happened ?

Reply:
@WeichenXu123 Following up from our offline discussion, it appears that the tests in this module are stuck because we're attempting to make an HTTP request against an invalid URL, and this request is being erroneously retried many times because we're specifying `total=None` and `connect=None` in our arguments to `urllib3.util.Retry`.

From the docs (https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html), the category of request failure encountered in the tests is a `connect`-related error. By default, `connect=None` when constructing Retry, implying that connect-related errors will be retried infinitely unless `total` is not `None`. Because we construct `Retry` with `total=None`, these requests are retried infinitely and our tests hang.